Maestro/subscriptions.json

```diff
@@ -160,6 +160,27 @@
             "/verbosity:Normal"
           ]
         },
+        // This handler will bring the Latest CoreFX release/1.1.0 build into CoreCLR release/1.1.0
```

Comment:
I am not sure I follow the comment - CoreCLR does not have a dependency on CoreFX, so why would we need this?